templates/beta/scripts/deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+cd ../../../aragonOS
+export ENS=$(npm run deploy:devnet:ens | tail -n 1) # get last line of output
+echo "Using ENS ${ENS}"
+npm run deploy:devnet:apm
+# extract and set ENS
+
+cd ../aragon-id
+npm run deploy:devnet
+
+cd ../aragon-apps
+env DEBUG=true npm run publish:devnet
```

Comment:
This is not working for me if I try to build (it worked when I used docker registry published image). I realized that there this line is commented out:
```
% docker run --env-file .env --network beta_devnet --network beta_ipfs --link ipfs aragon/templates-beta cat scripts/deploy.sh
#!/bin/bash

set -e

cd ../../../aragonOS
export ENS=$(npm run deploy:devnet:ens | tail -n 1) # get last line of output
echo "Using ENS ${ENS}"
npm run deploy:devnet:apm
# extract and set ENS

cd ../aragon-id
npm run deploy:devnet

cd ../aragon-apps
# env DEBUG=true npm run publish:devnet

cd templates/beta
npm run deploy:devnet
```